npm install fails

Problem with the same module as #158 
worked before #157 
```
npm i
npm WARN deprecated @types/file-type@10.9.1: This is a stub types definition. file-type provides its own type definitions, so you do not need this installed.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated @taku.moe/utils@1.0.9: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated @taku.moe/pura@1.0.4: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated @triptyk/tsyringe@4.7.1: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
npm ERR! code 7
npm ERR! path /home/otiskujawa/Documents/git/Naokoc/node_modules/gl
npm ERR! command failed
npm ERR! command sh -c -- prebuild-install || node-gyp rebuild
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@7.1.2
npm ERR! gyp info using node@20.2.0 | linux | x64
npm ERR! gyp info find Python using Python version 3.11.3 found at "/usr/bin/python3"
npm ERR! gyp ERR! UNCAUGHT EXCEPTION 
npm ERR! gyp ERR! stack TypeError: Cannot assign to read only property 'cflags' of object '#<Object>'
npm ERR! gyp ERR! stack     at createConfigFile (/home/otiskujawa/Documents/git/Naokoc/node_modules/node-gyp/lib/configure.js:117:21)
npm ERR! gyp ERR! stack     at /home/otiskujawa/Documents/git/Naokoc/node_modules/node-gyp/lib/configure.js:84:9
npm ERR! gyp ERR! stack     at FSReqCallback.oncomplete (node:fs:188:23)
npm ERR! gyp ERR! System Linux 6.3.5-arch1-1
npm ERR! gyp ERR! command "/home/otiskujawa/Documents/git/Naokoc/node_modules/node/bin/node" "/home/otiskujawa/Documents/git/Naokoc/node_modules/.bin/node-gyp" "rebuild"
npm ERR! gyp ERR! cwd /home/otiskujawa/Documents/git/Naokoc/node_modules/gl
npm ERR! gyp ERR! node -v v20.2.0
npm ERR! gyp ERR! node-gyp -v v7.1.2
npm ERR! gyp ERR! Node-gyp failed to build your package.
npm ERR! gyp ERR! Try to update npm and/or node-gyp and if it does not help file an issue with the package author.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/otiskujawa/.npm/_logs/2023-06-22T10_23_36_889Z-debug-0.log
```

--- a/README.md
+++ b/README.md
@@ -14,15 +14,15 @@ Next, depending on your platform:
 sudo apt install -y build-essential g++-10 libxi-dev libxext-dev libpixman-1-dev libcairo2-dev libpango1.0-dev libjpeg8-dev libgif-dev libjpeg-dev librsvg2-dev mesa-common-dev
 ```
 
-2. _TODO: set up mongo on linux_
-3. Use node.js 16.6.0 `nvm install 16.6.0 && nvm use 16.6.0`
-4. `npm i` to install dependencies
+2.  _TODO: set up mongo on linux_
+3.  install npm and yarn
+4.  `yarn` to install dependencies
 
 ### Windows
 
-1. Install Node version 16.x.x
+1. Install Node ~~version 16.x.x~~
 2. Install MongoDB Community https://www.mongodb.com/try/download/community & follow default installation
-3. With administrator privileges, do `npm i` in the repository folder
+3. With administrator privileges, do ~~`npm i`~~ `yarn` in the repository folder
 
 Once all these are set up:
 
@@ -32,10 +32,10 @@ Once all these are set up:
    Set the chat log ID
    Set url to mongodb (mongodb://127.0.0.1:27017 if hosting locally)
 2. In `constants/index.ts` replace guild & channel IDs with those of your test guild
-3. `npm run dev`
+3. `yarn dev`
 
 ### Docker
-
+_TODO: update if needed
 If you have docker installed you can simple build the image and run the install from inside the docker container
 ```shell
 docker compose run --rm -it node-cli yarn install

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "naoko",
-  "version": "2.1.4",
+  "version": "2.1.5",
   "description": "Naoko is Geoxor's Discord Bot",
   "exports": "./src/naoko.ts",
   "scripts": {
@@ -49,7 +49,7 @@
     "multi-progress": "^4.0.0",
     "music-metadata": "^8.1.4",
     "node": "20.2.0",
-    "node-canvas-webgl": "^0.2.7",
+    "@otiskujawa/node-canvas-webgl": "^0.2.7",
     "pet-pet-gif": "^1.0.9",
     "progress": "^2.0.3",
     "rcon.js": "^1.0.0",

--- a/src/service/3dProcessorService.ts
+++ b/src/service/3dProcessorService.ts
@@ -2,7 +2,7 @@ import { fileTypeFromBuffer } from "file-type";
 import { GifFrame, GifUtil } from "gifwrap";
 import Jimp from "jimp";
 // @ts-ignore this doesn't have types :whyyyyyyyyyyy:
-import { createCanvas, NodeCanvasElement } from "node-canvas-webgl";
+import { createCanvas, NodeCanvasElement } from "@otiskujawa/node-canvas-webgl";
 import * as THREE from "three";
 import comicSans from "../assets/comic_sans_font.json" assert { type: "json" };
 import cache from "../naoko/Cache";

